lib/jazz_fingers/configuration.rb

```diff
@@ -39,10 +39,10 @@ def awesome_print?
     end
 
     def application_name
-      return "(#{underscore(@application_name)})" unless @application_name.nil?
-      return "(#{Rails.application.class.parent_name.underscore})" if defined?(Rails)
+      return underscore(@application_name) unless @application_name.nil?
+      return Rails.application.class.parent_name.underscore if defined?(Rails)
```

Comment:
I like to have `()` around the project name to have a separated context.

Why the change?

Reply:
There's no change in behavior / prompt formatting — the parentheses are just moved to the prompt class to reduce duplication:

```rb
# lib/jazz_fingers/prompt.rb L48-49 (2453e7e2)

  blue_text("(#{name})")
end
```
<sup>[[source](https://github.com/jmromer/jazz_fingers/blob/2453e7e2/lib/jazz_fingers/prompt.rb#L48-L49)]</sup>